.claude/skills/gh-issue-writer/SKILL.md

```
@@ -19,40 +20,43 @@ Use this skill when the user wants to:
 ### Step 1: Parse User Input
 
 1. Determine issue type from prefix:
-   - `feat:` → Feature request
-   - `bug:` → Bug report
+  - `feat:` → Feature request
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)